### DISCUSSION TITLE
Copilot giving limits even in student developer pack

### DISCUSSION THREAD

**Discussion by Sarthak950** [community/community#147419] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have github student developer pack that comes with copilot pro subscription 
even after that it is showing me limit reached for the month
![image](https://github.com/user-attachments/assets/ae71a587-975d-4c80-b414-92b5fbb79a50)

even though 
![image](https://github.com/user-attachments/assets/29bb6916-6979-4a70-a6ad-9ba5c1a7fed5)

I am using this in neovim

Is this an issue or there is an hard limit for developer pack users 
Thanks

**Comment by FHitzenberger** [comment]
I just had the same problem with QT Creator 
![Screenshot 2024-12-18 184419](https://github.com/user-attachments/assets/ca518966-8064-4828-9777-cf85fca29bc0)

**Comment by Kin-Zhang** [comment]
same here: https://github.com/orgs/community/discussions/147420

![image](https://github.com/user-attachments/assets/3e6f8dae-e10e-49b4-a307-bccd14c0e8f5)

**Comment by barnabehvrd** [comment]
Same issue as a student with Webstorm since a few minutes 

![image](https://github.com/user-attachments/assets/0cafe5d0-b9c8-4cb3-a43a-d650710ec29d)
![image](https://github.com/user-attachments/assets/52d2b547-ec84-4a64-9467-2d78826845dc)

**Comment by Sarthak950** [comment]
maybe they have changed their policies or some bug lets see what happens

**Comment by tuves** [comment, marked as answer]
Update:
We’ve identified an issue affecting complimentary access to GitHub Copilot Pro for students, teachers and maintainers. We’re actively working on a fix. While we introduced a new free tier of Copilot today, your access to GitHub Copilot Pro will not change.

----

We're aware of this issue and working on it right now. Thanks for letting us know!

**Reply by avni** [reply]
Issue now resolved! Thanks for being patient with us. Continue enjoying your complimentary access to GitHub Copilot Pro! ❤️

**Reply by fvandrad** [reply]
Será que tem alguma relação do que ocorre comigo hoje?
O último uso do Copilot no VSCode foi em dezembro de 2024.
![Captura de tela_18-1-2025_17718_github com](https://github.com/user-attachments/assets/b1c1b031-95d3-46fb-b8e1-ad6ade49d2e9)

**Reply by MAlazhariy** [reply]
I'm facing exactly the same issue. I'm gonna report an issue now.

**Reply by MAlazhariy** [reply]
Check the issue I have filed an issue here on the same issue you are facing
Here is the link, I hope you support me to make them solve this issue
https://github.com/orgs/community/discussions/149578

**Reply by fvandrad** [reply]
It's working again now

**Reply by abdibehzad96** [reply]
I'm having the same issue now. It happened when my student package expired, and after I reapplied, it had limited access.

**Comment by Sarthak950** [comment]
@tuves thanks for the response

**Comment by Uzithei** [comment]
Same Issue with my account too @tuves

**Reply by niewiemczego** [reply]
Check that out https://github.com/orgs/community/discussions/149578#discussioncomment-12069067

**Reply by abdibehzad96** [reply]
You should contact their customer support; it worked for me.

**Comment by JacobHelwig** [comment]
I had Education benefits:
![image](https://github.com/user-attachments/assets/d6b4e79f-a4e0-4c98-b554-428bec61ac4c)
but not Copilot Pro. This answer solved it: https://github.com/orgs/community/discussions/158175#discussioncomment-13014521

You have to go to https://github.com/github-copilot/free_signup

**Reply by brun0duarte** [reply]
Thank you!

**Reply by LeLunZ** [reply]
Omg. That just solved my issue with GitHub copilot. After clicking the link, and signing up, even though I already have student benefits, I needed to logout and login again in the IDE and then everything worked!

**Comment by khalid35-877-ai** [comment]
<img width="1761" height="838" alt="Screenshot 2026-05-17 102117" src="https://github.com/user-attachments/assets/b293095f-c952-4bbd-97f1-fc8d77931890" />
<img width="1213" height="575" alt="Screenshot 2026-05-17 102301" src="https://github.com/user-attachments/assets/f6c6bb00-8463-47ff-91d4-f7cbcafafce8" />
Even though I have a student pack (pro) activated I see the same issues where my account is treated as a free account.

**Comment by Karthik-04** [comment]
@tuves - i am too facing this issue where I have github student developer pack that comes with copilot pro subscription
even after that it is showing me limit reached for the month.

Kindly help me fixing this issue asap.

<img width="1347" height="761" alt="image" src="https://github.com/user-attachments/assets/e3d11891-8c79-482c-a95f-ebfac7c8ef2b" />

**Reply by admin** [reply]
@Karthik-04  -- We have provided more details and a FAQ about usage-based billing in [this discussion](https://github.com/orgs/community/discussions/197089), which may be updated if new information becomes available. 

We had also provided more details about changes to Copilot Individual Plans in [this discussion](https://github.com/orgs/community/discussions/192963).

From the FAQ:
> 10. I'm newly eligible for Copilot Student but have never activated it. Can I sign up now?
At this time, new activations of Copilot Student, Pro, and Pro+ are paused.


> Your GitHub Education verification is active and has not been removed. The pause affects the ability to activate Copilot Student or a Copilot paid plan. When signups resume, you will be able to activate the Education benefit if you are still eligible. In the meantime, Copilot Free remains [available](https://github.com/features/copilot/plans) for signups.

> We do not have an estimated timeframe to share yet for when the pause will end.

As no other Community Discussions team members will be able to provide additional help, **and you are commenting on an old and resolved discussion, I am going to close and lock this discussion**. We appreciate your understanding.

To keep conversations all in one place and to ensure you receive timely updates, we recommend you comment on or subscribe to the discussions linked above.
